classes/xt_rcar.bbclass

```diff
@@ -0,0 +1,8 @@
+rcar_unpack_proprietary_graphics() {
```

Comment:
Correction: inc files are not visible in classes directory w/o altering BBPATH variable
so, instead I'll put it into inc/xt_rcar.inc